### ISSUE TITLE
Bug: Page number in TOC appears on two pages when close to page break

### ISSUE THREAD

**Issue by nzakas** [pagedjs/pagedjs#72]
**PagedJS Version:** v0.2.0

I've been using this [CSS for a TOC](https://pagedjs.org/post/toc/), and I'm seeing a duplicate page number for the last item on a page:

![toc-bug](https://user-images.githubusercontent.com/38546/163013094-9f4bee9c-e83c-4093-99c0-4a3916192aff.png)

I double-check that the HTML is being generated correctly. This happens regardless of which item ends up last on the page.

**Comment by julientaq** [comment]
hi @nzakas 
That happens sometimes when pseudo elements get rendered before the element itself. Would you mind sharing your HTML too so i can test it? 

Do you have a `break-inside:avoid` for the `li`?

**Comment by nzakas** [comment]
Here is a quick demo that doesn't quite work online, but contains the HTML and CSS I'm using:
https://printcss.live/KGRb4z2eBL

I continued playing and found if I added this to my CSS, then I could avoid the problem:

```css
.toc ol .toc-list-item {
    position: relative;
}
```

I didn't have `break-inside:avoid`! I must have copied-pasted incorrectly. It did fix the problem, too. 👍 

Thanks so much for your help.
